src/client-functions/selectors/add-api.js

```diff
@@ -268,6 +269,86 @@ function createDerivativeSelectorWithFilter (getSelector, SelectorBuilder, selec
     return builder.getFunction();
 }
 
+var ensureRegExpValue = str => typeof str === 'string' ? new RegExp(escapeRe(str)) : str;
+
+function sandboxed (fn) {
```

Comment:
You don't need manual sandboxing. ClientFunctions and their dependencies are automatically sandboxed.